### ISSUE TITLE
RDX: Unable to install some extensions

### ISSUE THREAD

**Issue by gunamata** [rancher-sandbox/rancher-desktop#4577]
I am not able to install some extensions on my Windows 10 machine. For example,

```
>rdctl extension install prakhar1989/dive-in:0.0.8
Internal Server Error
{"message":"500 Internal Server Error"}
```

At present, I am on this latest CI build - https://github.com/rancher-sandbox/rancher-desktop/actions/runs/4824251305
I am using `dockerd`  as the container engine.

Here are the logs from my session at this state:

[logs-20230428.zip](https://github.com/rancher-sandbox/rancher-desktop/files/11356011/logs-20230428.zip)

**Comment by gunamata** [comment]
Looking at the `extension.log`, it seems to be failing while running the command

```
docker cp --follow-link 2ff0f9bc68f5a0db08528ab95722a4bca6f7ca2fd2dc3bd90f1a2eba67fa4440:/. C:\\Users\\GunasekharMatamalam\\AppData\\Local\\rancher-desktop\\extensions\\cHJha2hhcjE5ODkvZGl2ZS1pbg\\compose
```

For some reason, the expected directory is not getting created on my machine?

**Comment by jandubois** [comment]
This issue was not part of the project and therefore never worked on. I've moved this to 1.10 and re-assigned back to @gunamata to verify if this is still an issue.

**Comment by gaktive** [comment]
Dive In shipped with 1.9.0 and works.
